### DISCUSSION TITLE
Problem with eda() function: name 'eda' is not defined (Colab and Jupyter Lab)

### DISCUSSION THREAD

**Discussion by tonpimenta** [pycaret/pycaret#3844] [Q&A] [answered]
Hello,
Can someone help me with this?
I'm trying to use eda() function right after I setup a classification experiment:
```
s = setup(data2pycaret, target = 'TARGET', session_id = 125,
          preprocess = True,
          feature_selection = True,
          fix_imbalance = True,
          normalize = True,
          categorical_features = cat_list,
          numeric_features = num_list,
          ordinal_features= None,
          max_encoding_ohe=25)
eda()
```
But I'm getting:
```
NameError                                 Traceback (most recent call last)
Cell In[11], line 1
----> 1 eda()

NameError: name 'eda' is not defined
```
Version:
pycaret==3.2.1
(Installed using pip install pycaret[full])

**Comment by tonpimenta** [comment, marked as answer]
I just found some information that eda() function is not supported anymore:
[here](https://github.com/pycaret/pycaret/issues/3614#issuecomment-1684409512)
is this correct?
If so, the function needs to be removed from documentation.

**Reply by moezali1** [reply]
@tonpimenta Where do u see EDA in documentation? Its already removed from pycaret.gitbook.io documentation.

**Reply by tonpimenta** [reply]
@moezali1 , thanks for your reply.
There's still a video here:
[EDA with pycaret](https://pycaret.gitbook.io/docs/learn-pycaret/videos#eda-with-pycaret)

**Reply by tonpimenta** [reply]
also here:
[pycaret.regression.eda](https://pycaret.readthedocs.io/en/latest/api/regression.html#pycaret.regression.eda)

**Comment by tom-flamelit** [comment]
eda() function still here as well.

https://pycaret.readthedocs.io/en/stable/api/classification.html#pycaret.classification.eda

**Reply by tom-flamelit** [reply]
Looks like readthedocs built has successfully built for awhile
